app/controllers/admin/users_controller.rb

```diff
@@ -21,7 +21,7 @@ def update
 
   # DELETE /admin/users/1
   def destroy
-    @user.destroy
+    @user.enrollments.where(team: current_team).delete_all
```

Comment:
hm, because enrollments is merely a many-to-many table with not much logic of its own, either are ok, I suppose. Do you see any edge-cases?